[2024-09-25T11:13:27.727+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T11:13:27.791+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.pipeline.warehouse.step_1.load.orders_history manual__2024-09-25T11:08:58.108660+00:00 [queued]>
[2024-09-25T11:13:27.824+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.pipeline.warehouse.step_1.load.orders_history manual__2024-09-25T11:08:58.108660+00:00 [queued]>
[2024-09-25T11:13:27.826+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-25T11:13:27.886+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): pipeline.warehouse.step_1.load.orders_history> on 2024-09-25 11:08:58.108660+00:00
[2024-09-25T11:13:27.905+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1075) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T11:13:27.909+0000] {standard_task_runner.py:72} INFO - Started process 1077 to run task
[2024-09-25T11:13:27.911+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'pipeline.warehouse.step_1.load.orders_history', 'manual__2024-09-25T11:08:58.108660+00:00', '--job-id', '261', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmpvo30qf65']
[2024-09-25T11:13:27.918+0000] {standard_task_runner.py:105} INFO - Job 261: Subtask pipeline.warehouse.step_1.load.orders_history
[2024-09-25T11:13:28.039+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.pipeline.warehouse.step_1.load.orders_history manual__2024-09-25T11:08:58.108660+00:00 [running]> on host e10075799413
[2024-09-25T11:13:28.303+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='pipeline.warehouse.step_1.load.orders_history' AIRFLOW_CTX_EXECUTION_DATE='2024-09-25T11:08:58.108660+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-25T11:08:58.108660+00:00'
[2024-09-25T11:13:28.306+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T11:13:28.341+0000] {base.py:84} INFO - Retrieving connection 'minio'
[2024-09-25T11:13:28.419+0000] {base.py:84} INFO - Retrieving connection 'warehouse_db'
[2024-09-25T11:13:32.900+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-25T11:13:32.918+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T11:13:32.919+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=pipeline.warehouse.step_1.load.orders_history, run_id=manual__2024-09-25T11:08:58.108660+00:00, execution_date=20240925T110858, start_date=20240925T111327, end_date=20240925T111332
[2024-09-25T11:13:32.971+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T11:13:33.069+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-25T11:13:33.073+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
